src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfonyLegacy.php

```diff
@@ -13,6 +13,9 @@
  * if the old symfony interface is detected.
  *
  * @deprecated Do not reference this class directly, use TestworkEventDispatcher
+ *
+ * @todo Remove this class in the next major version. It's not used anymore, but just left here
+ *       for BC purposes.
```

Comment:
add some `trigger_error` instead of such comments

Reply:
Good call, I've removed the comment and added some `trigger_error` with `E_USER_DEPRECATED` 👍